src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputSkipIfLastAndSharedWithCase.java

```diff
@@ -0,0 +1,113 @@
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+
+public class InputSkipIfLastAndSharedWithCase
+{
+    void method(int i) {
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+                break;
+            case 2:
+                break;
+        }
+
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+            case 2:
+                break;
+            case 3:
+                break;
+        }
+
+        switch (i) {
+            default: // No violation with the new option is expected
```

Comment:
@sagar-shah94 Look at romani's post at https://github.com/checkstyle/checkstyle/issues/4078#issuecomment-289211757 .
> default: // violation !!!! with the new option is expected

There should be a violation here by his example. `default` should be requested to be on bottom.
The current message may not make sense for this new violation.
> Default should be last label in the switch.

I am thinking it should be `Default should be last label in the case group`.

@romani Please confirm.
Please also confirm, logic of check should not change if `skipIfLastAndSharedWithCase` is turned off? It should behave as it does now. Only changes are what you specified, suppression of one case and new violation in other.